XCUITests/FxScreenGraph.swift

```diff
@@ -638,6 +640,9 @@ func createScreenGraph(for test: XCTestCase, with app: XCUIApplication) -> MMScr
         screenState.gesture(forAction: Action.SelectNewTabAsBlankPage) { UserState in
             table.cells["Blank Page"].tap()
         }
+        screenState.gesture(forAction: Action.SelectNewTabAsFirefoxHomePage) { UserState in
+            table.cells["Firefox Home"].tap()
```

Comment:
I have been checking how to add ids for this elements so that we better use ids instead of labels (we should try that whenever elements are added to the ScreenGraph) and looks easy, would you mind to do it in this PR? 
Just go to this file and add each id like FirefoxHome for this case in [this line](https://github.com/mozilla-mobile/firefox-ios/blob/dec2afc11d9ad59a3081fc095952eee5a977e1ed/Client/Frontend/Settings/NewTabContentSettingsViewController.swift#L36) instead of nil in: `accessibilityIdentifier: nil`
And the same for the rest, Blank Page, Bookmarks, History. For custom url looks like there is an id already `HomePageSetting`

Let me know if you have any questions about that.

Reply:
I have done this in the latest commit. I am also home to replace the ID 'HomePageSetting' with a more descriptive ID- 'HomePageSetting' doesn't seem to represent setting a Custom URL for Home or Tab very well. This may require some troubleshooting on my part to make sure nothing breaks as I have renamed it. So far, nothing has seemed to break.